docs/ref/booking/notes.rst

```diff
@@ -0,0 +1,19 @@
+booking/notes
+--------------
+This is a call to your Checkfront account that pulls all the notes from all of your bookings within a certain time frame. By default this will pull all notes from all your bookings in the system. 
```

Comment:
Should be something more like:
This endpoint return all of the notes from your bookings. By default it will return the notes for all booking dates, but you can filter the notes by a specific time frame.